modules/gltf/structures/gltf_skin.cpp

```diff
@@ -205,18 +204,18 @@ Error GLTFSkin::from_dictionary(const Dictionary &dict) {
 	ERR_FAIL_COND_V(!dict.has("joint_i_to_bone_i"), ERR_INVALID_DATA);
 	Dictionary joint_i_to_bone_i_dict = dict["joint_i_to_bone_i"];
 	joint_i_to_bone_i.clear();
-	for (int i = 0; i < joint_i_to_bone_i_dict.keys().size(); ++i) {
-		int key = joint_i_to_bone_i_dict.keys()[i];
-		int value = joint_i_to_bone_i_dict[key];
+	for (const KeyValue<Variant, Variant> &kv : joint_i_to_bone_i_dict) {
```

Comment:
Oof... This one was re-creating the keys array 3 times each iteration. That really hurts.